CHANGELOG.md

```diff
@@ -125,6 +125,7 @@ This release contains the [PDV refactor (#4111)](https://github.com/hasura/graph
 - server: fix issue with tracking custom functions that return `SETOF` materialized view (close #5294) (#5945)
 - server: introduce optional custom table name in table configuration to track the table according to the custom name. The `set_table_custom_fields` API has been deprecated, A new API `set_table_customization` has been added to set the configuration. (#3811)
 - server: allow remote relationships with union, interface and enum type fields as well (fixes #5875) (#6080)
+- server: clean up the post-PDV schema generation (#6116)
```

Comment:
I feel we do not need a changelog for this as it is kinda vague and definitely not user understandable.

Reply:
Perfect. I was on the fence myself.